laravel/app/Http/Controllers/InstanceController.php

```diff
@@ -32,15 +32,10 @@ public function overview(): View
                 $virtualserver_helper = new TeamSpeakVirtualserver($instance);
                 $virtualserver = $virtualserver_helper->get_virtualserver_connection();
                 $channel_list = $virtualserver->channelList();
+                $channelListForEachInstance[$instance->id] = $channel_list;
             } catch (ServerQueryException $serverquery_exception) {
-                return view('instance.edit', ['instance_id' => $instance->id])->with([
-                    'error' => 'instance-channellist-error',
-                    'message' => $serverquery_exception->getMessage(),
-                    'instance' => $instance,
-                    'channel_list' => [],
-                ]);
+                $channelListForEachInstance[$instance->id] = ['error' => $serverquery_exception->getMessage()];
```

Comment:
[SUGGESTION]

As mentioned above and to simplify the code a bit:
```suggestion
                $channelListForEachInstance[$instance->id]['channel_list'] = [];
                $channelListForEachInstance[$instance->id]['error'] = $serverquery_exception->getMessage();
```